src/Windowing/OpenToolkit.Windowing.Common/Interfaces/INativeWindow.cs

```diff
@@ -53,5 +53,39 @@ public interface INativeWindow : INativeWindowProperties, IWindowEvents, IWindow
         /// The point transformed to screen coordinates.
         /// </returns>
         Vector2i PointToScreen(Vector2i point);
+
+        /// <summary>
+        /// Gets the current monitor scale.
+        /// </summary>
+        /// <returns><c>true</c>, if current monitor scale was gotten correctly, <c>false</c> otherwise.</returns>
```

Comment:
You should probably switch this around so the returns is at the bottom of the xml comments, like the other comments.